packages/cli/src/commands/push.ts

```diff
@@ -14,32 +14,32 @@ import {
   BundleOutputFormat,
   getTotals,
   slash,
+  Region,
 } from '@redocly/openapi-core';
 import {
-  promptUser,
   exitWithError,
   printExecutionTime,
   getFallbackEntryPointsOrExit,
   pluralize,
   dumpBundle,
 } from '../utils';
+import { promptClientToken } from './login';
+import { getRawConfigContent } from '../../../core/src/config/load';
 
 export async function handlePush(argv: {
   entrypoint?: string;
   destination?: string;
   branchName?: string;
   upsert?: boolean;
   'run-id'?: string;
+  region?: Region;
 }) {
-  const client = new RedoclyClient();
+  const region = argv.region || (await getRawConfigContent()).region;
+  const client = new RedoclyClient(region);
   const isAuthorized = await client.isAuthorizedWithRedocly();
+
   if (!isAuthorized) {
-    const clientToken = await promptUser(
-      green(
-        `\n  🔑 Copy your API key from ${blue('https://app.redoc.ly/profile')} and paste it below`,
-      ),
-      true,
-    );
+    const clientToken = await promptClientToken(client.getDomain());
```

Comment:
it looks similar to https://github.com/Redocly/openapi-cli/pull/459/files#diff-fa0a4d0e8d3c6b6651500902a739e43cfa0935d9d5c9ab939e62032ba1588e3dR16
why do not reuse it?

Reply:
thx, will do if it reasonable